llarp/service/outbound_context.cpp

```diff
@@ -567,7 +567,7 @@ namespace llarp
         // verify source
         if (!frame.Verify(si))
         {
-          LogWarn("signature failed");
+          LogWarn("signature failed, T=", frame.T);
```

Comment:
should be "signature verification failed", right?  Also perhaps instead of `T=` put a descriptive name.